nemo/collections/asr/data/audio_to_text.py

```diff
@@ -20,11 +20,10 @@
 from nemo.core.neural_types import *
 from nemo.utils.decorators import experimental
 
-__all__ = ['AudioToTextDataset']
+__all__ = ['AudioToCharDataset', 'AudioToBPEDataset']
```

Comment:
Should we typehint `AudioToCharDataset` and `AudioToBPEDataset`?

Reply:
Yes we should, forgot about that.

Reply:
Done